### PR TITLE
V1 Prototype Documentation 

### PR DESCRIPTION
**Pull request by Olshansk**
## Description

This PR is meant to cleanup documentation related to the V1 prototype in order to close out the [V1 Milestone Protoype](https://github.com/pokt-network/pocket/milestone/1). It fixes a few minor bugs, but also refactors and updates the documentation so it is easier to follow.

**Fixes issues**
* https://github.com/pokt-network/pocket/issues/21
* https://github.com/pokt-network/pocket/issues/20

**Other fixes**
This change also fixes some issues (protobuf, configuration, etc) that were introduced in the last stages of the integration phase. Anyone cloning the repository anew should be able to run a localnet by following the instructions.

## Type of change
Please mark the options that are relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?
This change was tested by following the new instructions added to `docs/development/README.md`.

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [ ] No new warnings are generated by my changes
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by andrewnguyen22 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,22 +6,44 @@
 
 # Pocket
 
-The official implementation of the [V1 Pocket Network Protocol](https://github.com/pokt-network/pocket-network-protocol)
+The official implementation of the [V1 Pocket Network Protocol Specification](https://github.com/pokt-network/pocket-network-protocol).
 
-## Documentation
+\*_Please note that V1 development is currently under development and see [pocket-core](https://github.com/pokt-network/pocket-core) for the version that is currently live on mainnet._\*
```
Comment (guettomusick) [inline README.md:11]:
```suggestion
\*_Please note that V1 protocol is currently under development and see [pocket-core](https://github.com/pokt-network/pocket-core) for the version that is currently live on mainnet._\*
```

File: shared/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        C <--> B
+        U <--> B
+        P2P <--> B
+        P <--> db
+        P2P <-.-> N((Pocket Network))
+        db[(Pocket Database)]
+    end
+
+    click P callback "Tooltip for a callback"
+    click P call callback() "Tooltip for a callback"
```
Comment (guettomusick) [inline shared/README.md:72]: Is this needed? Doesn't seem to do anything

**Review by guettomusick — APPROVED** [review]
LGTM, on the documentation changes.

File: shared/README.md (hunk 2 above)
Comment (Olshansk) [reply shared/README.md:72]: It is but I forgot to finish it. Fixed!

**Pull request merged**